src/main/java/spoon/Launcher.java

```diff
@@ -485,6 +493,14 @@ protected void processArguments() {
 	}
 
 	/**
+	 * Gets the list of processors instance to be initially applied during the
+	 * processing.
+	 */
+	protected List<Processor<? extends CtElement>> getProcessorsInstance() {
```

Comment:
Why not call it getProcessors()?...
Is the access modifier with protected the right one? You could make it public and either return a unmodifiable list or an array.